File: snippets/py/array/empty/empty.py
array = []
if (not len(array)): print('array is empty')  # ≡
array.append(100)  # add an item
if (len(array)): print('array has items')     # ≡
del array[:]  # clear the array
if (not len(array)): print('array is empty again')

File: snippets/rb/array/empty/empty.rb
array = []
if (not array.length()) 
    print("array is empty")  # ≡
end

array.append(100)  # add an item
if (array.length())
    print("array has items\n")     # ≡
end

array.clear()

if (array.length() == 0)
    print("array is empty again\n")
end